devenv/docker/blocks/mysql_tests/docker-compose.yaml

```diff
@@ -3,6 +3,7 @@
       context: docker/blocks/mysql_tests
       args:
         - mysql_version=${mysql_version}
+    platform: linux/amd64
```

Comment:
This is necessary to get the MySQL image running on M-series Macs because there's no arm64 package for the MySQL image we use.